config/stage.json

```diff
@@ -23,6 +23,7 @@
 		"happychat": true,
 		"help": true,
 		"help/courses": true,
+		"help/directly": true,
```

Comment:
We typically don't enable features in staging until the feature is pretty much ready for launch, to keep staging and production as similar as possible.

The [wpcalypso](https://wpcalypso.wordpress.com) and [Horizon](https://horizon.wordpress.com) environments make for good testing grounds for incomplete features in the meantime.